aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/AWSS3StoragePlugin.java

```diff
@@ -105,6 +107,7 @@ public final class AWSS3StoragePlugin extends StoragePlugin<S3Client> {
      * Constructs the AWS S3 Storage Plugin initializing the executor service.
      */
     @SuppressWarnings("unused") // This is a public API.
+    @OptIn(markerClass = InternalApiWarning.class)
```

Comment:
Does this mean creating plugins will throw internal API warning? Both the methods are intended for public use.

Reply:
The `OptIn` means that the warning won't be presented to callers of these methods.